.github/workflows/doc.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       run: sudo apt-get install pandoc
     - name: Generate doc
       run: |
-        cd docs && make venv-doc && cp -R _build/html ~/html && cd ..
+        cd docs && make venv-doc && make doctest && cp -R _build/html ~/html && cd ..
```

Comment:
I think it needs to run in the right env right ? Better adapt the venv-doc to make it happen